src/utils/db/db.ts

```diff
@@ -496,6 +496,49 @@ export class DB {
         return person
     }
 
+    public async setPersonAsIdentified(person: Person): Promise<void> {
+        let shouldUpdateClickhouse = false
+
+        const result = await this.postgresQuery(
+            `
+                UPDATE posthog_person 
+                SET is_identified=$1 
+                WHERE id = $2 
+                AND is_identified = $3
+                RETURNING is_identified
+            `,
+            [true, person.id, false],
+            'setPersonAsIdentified'
+        )
+
+        if (result.rows.length > 0) {
+            shouldUpdateClickhouse = true
+        }
```

Comment:
nit in python I'd do (& drop it from the top) not sure if valid ts
```
let shouldUpdateClickhouse = result.rows.length > 0
```

Reply:
ah yes ofc this lingered from before I had the const

Reply:
addressed, cheers